README.md

```diff
@@ -1,13 +1,27 @@
 # SIMPLE
 
-The vision of the SIMPLE project is to create a *collaborative* database of low-mass stars, brown dwarfs, and directly imaged exoplanets: a simple archive of complex objects. We plan to include coordinates, photometry, kinematics, spectra, images. We are also considering ways to include modeled and retrieved parameters. We are developing methods to interact with the database via several methods, including python, a website and API, and database browsers.
+The vision of the SIMPLE project is to create a *collaborative* database of low-mass stars, brown dwarfs, and directly 
+imaged exoplanets: a simple archive of complex objects. The tables and fields currently included in the 
+database are described in the [Documentation](https://github.com/SIMPLE-AstroDB/SIMPLE-db/blob/main/documentation/README.md) 
```

Comment:
I would suggest we use relative paths (see, for example, the way the image is done) rather than absolute http links. That also allows for different documentation across branches, in case that situation comes up.